apps/admin-gui/src/app/shared/side-menu/side-menu-item.service.ts

```diff
@@ -237,6 +217,24 @@ export class SideMenuItemService {
           url: ['/admin/blocked_logins'],
           activatedRegex: '^/admin/blocked_logins',
         },
+        {
+          label: 'MENU_ITEMS.ADMIN.VISUALIZER',
+          url: ['/admin/visualizer'],
+          activatedRegex: '^/admin/visualizer$',
+          children: [
+            {
+              label: 'MENU_ITEMS.VISUALIZER.ATTR_DEPENDENCIES',
+              url: ['/admin/visualizer/attrDependencies'],
+              activatedRegex: '^/admin/visualizer/attrDependencies',
+            },
+            {
+              label: this.translate.instant('MENU_ITEMS.VISUALIZER.USER_DESTINATION') as string,
```

Comment:
Can you please replace it with `PerunTranslateService` and remove `as string`.

Reply:
Removed redundant translation.